Sources/Sentry/SentryClient.m

```diff
@@ -159,7 +159,7 @@ - (SentryId *)captureError:(NSError *)error
 - (SentryEvent *)buildErrorEvent:(NSError *)error
 {
     SentryEvent *event = [[SentryEvent alloc] initWithLevel:kSentryLevelError];
-    event.message = error.localizedDescription;
+    event.message = [NSString stringWithFormat:@"%@ %ld", error.domain, (long)error.code];
```

Comment:
L: could this boxing fail?

Reply:
Good point. `error.code` is a NSInteger which can be both a 32-bit or 64-bit integer. [The official way of Apple](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Strings/Articles/formatSpecifiers.html) is to cast an NSInteger to a `long` and use `%ld`. So this is fine then. 
We could also use `%zd`, but I prefer to go with the Apple way. In case you are interested in this topic, this is a good summary: https://useyourloaf.com/blog/format-string-issue-using-nsinteger/